src/scores/continuous/standard_impl.py

```diff
@@ -418,3 +419,147 @@ def pbias(
+        - :math:`\\alpha` is the ratio of the standard deviations (variability ratio)
+        - :math:`\\beta` is the ratio of the means (bias)
+        - :math:`x` and :math:`y` are forecast and observed values, respectively
+        - :math:`\\mu_x` and :math:`\\mu_y` are the means of forecast and observed values, respectively
+        - :math:`\\sigma_x` and :math:`\\sigma_y` are the standard deviations of forecast and observed values, respectively
+        - :math:`s_\\rho`, :math:`s_\\alpha` and :math:`s_\\beta` are the scaling factors for the correlation coefficient :math:`\\rho`,
+            the variability term :math:`\\alpha` and the bias term :math:`\\beta`
+
+    Args:
+        fcst: Forecast or predicted variables.
+        obs: Observed variables.
+        reduce_dims: Optionally specify which dimensions to reduce when
+            calculating the KGE. All other dimensions will be preserved.
+        preserve_dims: Optionally specify which dimensions to preserve when
+            calculating the KGE. All other dimensions will be reduced. As a
+            special case, 'all' will allow all dimensions to be preserved. In
+            this case, the result will be in the same shape/dimensionality
+            as the forecast, and the errors will be the error at each
+            point (i.e. single-value comparison against observed), and the
+            forecast and observed dimensions must match precisely.
+        scaling_factors : A 3-element vector or list describing the weights for each term in the KGE.
+            Defined by: scaling_factors = [s_rho, s_alpha, s_beta] to apply to the correlation (rho),
+            variability (alpha), and bias (beta) terms respectively. Defaults to (1.0, 1.0, 1.0). (*See
+            equation 10 in Gupta et al. (2009) for definitions of s_rho, s_alpha and s_beta*)
+        return_components (default False): If True, the function also returns the individual terms contributing to the KGE score:
+
+            - `rho`: Pearson correlation coefficient between forecasts and observations.
+            - `alpha`: Variability ratio (forecast/observation standard deviation).
+            - `beta`: Bias ratio between forecast and observation mean.
+
+    Returns:
+        The Kling-Gupta Efficiency (KGE) score as an xarray DataArray.
+
+        If ``return_components`` is True, the function returns ``xarray.Dataset`` kge_s with the following variables:
+
+        - `kge`: The KGE score.
+        - `rho`: The Pearson correlation coefficient.
+        - `alpha`: The variability ratio.
+        - `beta`: The bias term.
+
+    Notes:
+        - Statistics are calculated only from values for which both observations and
+          simulations are not null values.
+        - This function isn't set up to take weights.
+        - Currently this function is working only on xarray data array
+        - preserve_dims: 'all' returns NaN like pearson correlation as std dev is zero for a single point
```

Comment:
```suggestion
        - preserve_dims: 'all' returns NaN like pearson correlation as std dev is zero for a single point
```
1. Could this dot point about preserve_dims could perhaps be clearer? 
2. Is "std dev" an abbreviation for "standard deviation." If so, it is clearer to use the full term, or is the abbreviation so commonly used it is clearer to use the abbreviation?